crates/hotshot/examples/infra/modDA.rs

```diff
@@ -313,6 +313,13 @@ pub trait RunDA<
                                 }
                             }
 
+                            // send transactions
+                            for _ in 0..(total_nodes_u64 / transactions_per_round as u64) {
```

Comment:
I'm wondering why it's `total_nodes_u64 / transactions_per_round`. E.g., if there are 10 nodes and it's 2 txn/round, do we submit transactions 5 times? I think it should be `total_transactions / transactions_per_round`, but maybe I'm misunderstanding something.

Reply:
Yes sorry, totally missed that . Fixed !